application/org.openjdk.jmc.flightrecorder.ui/src/main/resources/org/openjdk/jmc/flightrecorder/ui/messages/internal/messages.properties

```diff
@@ -347,6 +347,8 @@ NAVIGATOR_ZOOM_OUT_TEXT=Zoom Out
 
 PREFERENCES_CONFIRM_REMOVE_TEMPLATE_TEXT=Ask for confirmation before removing flight recording templates
 PREFERENCES_ENABLE_RECORDING_ANALYSIS=Enable flight recording analysis
+PREFERENCES_WEBSOCKET_SERVER_PORT_TEXT=Websocket server port (0 - disabled)
```

Comment:
Perhaps (set to 0 to disable) or (0 means disabled)? The dash made me look twice.